app/code/Magento/Tax/Model/Calculation/RateRepository.php

```diff
@@ -102,11 +102,14 @@ public function __construct(
         $this->regionFactory = $regionFactory;
         $this->resourceModel = $rateResource;
         $this->joinProcessor = $joinProcessor;
-        $this->collectionProcessor = $collectionProcessor ?: $this->getCollectionProcessor();
+        $this->collectionProcessor = $collectionProcessor ?: \Magento\Framework\App\ObjectManager::getInstance()->get(
+        // phpstan:ignore "Class Magento\Tax\Model\Api\SearchCriteria\TaxRateCollectionProcessor not found."
+            \Magento\Tax\Model\Api\SearchCriteria\TaxRateCollectionProcessor::class
+        );
```

Comment:
Hello @SilinMykola 
Please use null-coalescing operator in constructor body for properties

```suggestion
        $this->collectionProcessor = $collectionProcessor ?? \Magento\Framework\App\ObjectManager::getInstance()->get(
        // phpstan:ignore "Class Magento\Tax\Model\Api\SearchCriteria\TaxRateCollectionProcessor not found."
            \Magento\Tax\Model\Api\SearchCriteria\TaxRateCollectionProcessor::class
        );
```

Reply:
Hello @andrewbess 
Thanks for your review. I made changes, please check.